Simplify Github issue and pull request templates

msulprizio opened the pull request:
The existing issue and pull request templates have been removed and replaced with templates that have been greatly simplified. The new templates:

1. Point users to the support and contribution guidelines, documentation, and debug pages first.
2. Include a section for users to identify themselves and their institution since this is not always obvious from their Github accounts.
3. Remove automatic tags and bracketed titles. These should be assigned by the user or by the GCST upon initial review of new issues.

The CONTRIBUTING.md and SUPPORT.md files have also been updated to include links to one another.

See also:
- https://github.com/geoschem/geos-chem/pull/1650
- https://github.com/geoschem/HEMCO/pull/195
- https://github.com/geoschem/GCClassic/pull/32

msulprizio committed:
Update Github issue templates

The existing issue and pull request templates have been removed and replaced
with templates that have been greatly simplified. The new templates:

1. Point users to the support and contribution guidelines, documentation, and
   debug pages first.
2. Include a section for users to identify themselves and their institution
   since this is not always obvious from their Github accounts.
3. Remove automatic tags and bracketed titles. These should be assigned by
   the user or by the GCST upon initial review of new issues.

The CONTRIBUTING.md and SUPPORT.md files have also been updated to include
links to one another.

Signed-off-by: Melissa Sulprizio <mpayer@seas.harvard.edu>

yantosca commented on .github/ISSUE_TEMPLATE/question-issue.md:
```diff
@@ -0,0 +1,21 @@
+
+### Confirm you have reviewed the following documentation
+
+- [ ] [Support guidelines](https://gchp.readthedocs.io/en/latest/reference/SUPPORT.html)https://geos-chem.readthedocs.io/en/stable/reference/SUPPORT.html)
```
The URL is listed twice here.

msulprizio replied:
Now resolved in https://github.com/geoschem/GCHP/pull/293/commits/e19cbea17e0e20ce0a9cc09ab9c0f75f8b317ed7.

yantosca replied:
Great, good to merge now!

yantosca reviewed (changes requested):
Looks good to merge after fixing the duplicate URL.

msulprizio committed:
Fix typo in Github issues template

The link for support guidelines in .github/ISSUE_TEMPLATE/question-issue.md
included an extra link that pointed to the geos-chem RTD.

Signed-off-by: Melissa Sulprizio <mpayer@seas.harvard.edu>
